engine/dlib/src/dmsdk/dlib/configfile.h

```diff
@@ -44,10 +44,18 @@ namespace dmConfigFile
      * Get config value as string, returns default if the key isn't found
      *
      * @name dmConfigFile::GetString
-     * @param config [type:dmBuffer::HConfig] Config file handle
+     * @param config [type:dmConfigFile::HConfig] Config file handle
      * @param key [type:const char*] Key in format section.key (.key for no section)
      * @param default_value [type:const char*] Default value to return if key isn't found
      * @return value [type:const char*] found value or default value
+     * @examples
+     *
+     * ```cpp
+     * static dmExtension::Result AppInitialize(dmExtension::AppParams* params)
+     * {
+     *     int displayWidth = dmConfigFile::GetString(params->m_ConfigFile, "display.width", 640);
+     * }
+     * ```
```

Comment:
The example is a bit odd.
int != const char* etc..

Reply:
This is the example for _GetString()_, and should probably not show how to get an integer.